apps/config/storage.py

```
@@ -0,0 +1,17 @@
+from whitenoise.storage import CompressedManifestStaticFilesStorage
+
+
+class ManifestStaticFilesStorage(CompressedManifestStaticFilesStorage):
+    # manifest_strict only guards stored_name() (runtime), not _stored_name()
+    # (post-processing). Override _stored_name() so missing source-map references
+    # from third-party packages don't abort collectstatic.
+    # TODO CT: Check if django-passkeys has fixed the missing bootstrap-toggle.min.js.map
+    #       and remove this class + revert settings.py to CompressedManifestStaticFilesStorage.
+    #       Tracking issue: https://github.com/mkalioby/django-passkeys/issues
+    def _stored_name(self, name, hashed_files):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_stored_name`

(ANN202)